MekHQ/data/universe/awards/standard.xml

```diff
@@ -0,0 +1,552 @@
+		<size>Individual</size>
+		<range>Career</range>
+	</award>
+	<award>
+		<name>Task Force Serpent Campaign</name>
+		<description>Partook combat actions in support of Task Force Serpent.</description>
+		<medal>TaskForceSerpentM.png</medal>
+		<ribbon>3-08-6-TaskForceSerpent.png</ribbon>
+		<xp>1</xp>
+		<position>3.08.6</position>
+		<qty>1</qty>
+		<item>TFS</item>
+		<size>Individual</size>
+		<range>Career</range>
+	</award>
+	<award>
+		<name>Galactic Service Deployment</name>
+		<description>Served over 3 consecutive months in a foreign theatre.</description>
+		<ribbon>4-01-2-GalacticServiceDeployment.png</ribbon>
+		<xp>1</xp>
+		<position>4.01.2</position>
+		<qty>3</qty>
+		<item>Foreign</item>
+		<size>Individual</size>
+		<range>Months</range>
+	</award>
+	<award>
+		<name>Galactic Service</name>
+		<description>Served a total of 12 months in foreign theatres.</description>
+		<ribbon>4-02-2-GalacticService.png</ribbon>
+		<xp>1</xp>
+		<position>4.02.2</position>
+		<qty>12</qty>
+		<item>Foreign</item>
+		<size>Individual</size>
+		<range>Months</range>
+	</award>
+	<award>
+		<name>Covert Ops</name>
+		<description>Served on a covert operations team.</description>
+		<ribbon>4-02-8-CovertOps.png</ribbon>
+		<xp>2</xp>
+		<edge>1</edge>
+		<position>4.02.8</position>
+		<qty>1</qty>
+		<item>Covert</item>
+		<size>Individual</size>
+		<range>Career</range>
+	</award>
+	<award>
+		<name>Longevitiy</name>
+		<description>Award for every 4 years of service.</description>
+		<ribbon>4-03-1-Longevity.png</ribbon>
+		<xp>4</xp>
+		<stackable>true</stackable>
+		<position>4.03.1</position>
+		<qty>48</qty>
+		<item>Time</item>
+		<size>Individual</size>
+		<range>Months</range>
+	</award>
+	<award>
+		<name>Drill Instructor</name>
+		<description>Served as basic training drill instructor.</description>
+		<ribbon>4-04-5-DrillInstructor.png</ribbon>
+		<xp>2</xp>
+		<edge>1</edge>
+		<position>4.04.5</position>
+		<qty>1</qty>
+		<item>Drill</item>
+		<size>Individual</size>
+		<range>Career</range>
+	</award>
+	<award>
+		<name>Military Training Instructor</name>
+		<description>Served as military instructor.</description>
+		<ribbon>4-04-6-MilitaryTrainingInstructor.png</ribbon>
+		<xp>1</xp>
+		<edge>1</edge>
+		<position>4.04.6</position>
+		<qty>1</qty>
+		<item>Instructor</item>
+		<size>Individual</size>
+		<range>Career</range>
+	</award>
+	<award>
+		<name>Ceremonial Duty</name>
+		<description>Served as honor guard.</description>
+		<ribbon>4-05-1-CeremonialDuty.png</ribbon>
+		<xp>1</xp>
+		<position>4.05.1</position>
+		<qty>1</qty>
+		<item>Honor</item>
+		<size>Individual</size>
+		<range>Career</range>
+	</award>
+	<award>
+		<name>Securty Guard</name>
+		<description>Served as security guard.</description>
+		<ribbon>4-05-2-SecurityGuard.png</ribbon>
+		<xp>1</xp>
+		<edge>1</edge>
+		<position>4.05.2</position>
+		<qty>1</qty>
+		<item>Security</item>
+		<size>Individual</size>
+		<range>Career</range>
+	</award>
```

Comment:
There is a real ribbon for those that serve on security forces.  IRL, it's a job for those that usually don't score high enough for placement in other positions.  It's an important job, but often thankless, so somewhere in the past the powers that were decided to give them an extra ribbon to wear.  In the interest of character background, I added it.  It will make more sense when you review the Awards.md and the Assignment Award Tables.